presto-docs/src/main/sphinx/release/release-0.254.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.254
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where queries that have both remote functions and a local function with only constant arguments could throw an IndexOutOfBoundException during planning. The bug was introduced in release 0.253 by :pr:`16039`.
+* Add fragment result caching support for ``UNNEST`` queries.
+* Add :func:`poisson_cdf` and :func:`inverse_poisson_cdf` functions.
+* Add memory tracking in ``TableFinishOperator`` which can be enabled by setting the ``table-finish-operator-memory-tracking-enabled`` configuration property to ``true``. Enabling this property can help investigating GC issues on the coordinator, by allowing us to debug whether stats collection uses a lot of memory.
+* Remove spilling strategy ``PER_QUERY_MEMORY_LIMIT`` and instead add configuration property ``experimental.query-limit-spill-enabled`` and session property ``query_limit_spill_enabled``. When ``query_limit_spill_enabled`` is set to ``true`` and the spill strategy is not ``PER_TASK_MEMORY_THRESHOLD``, then we will spill whenever a query uses more than the per-node total memory limit in combined revocable and non-revocable memory.  We will also still spill whenever the memory pool exceeds the spill threshold. This fixes an issue where using the ``PER_QUERY_MEMORY_LIMIT`` spilling strategy could prevent the oom killer from running when the memory pool was full. The issue is still present for the ``PER_TASK_MEMORY_THRESHOLD`` spilling strategy.
+
+Hive Changes
+____________
+* Add support for MaxResults on Glue Hive Metastore. MaxResults is the maximum number of partitions to return in a single response.
```

Comment:
@v-jizhang I'm still very confused about this release note.  It seems like what the PR (https://github.com/prestodb/presto/pull/16012) actually does is set the batch size for how many partitions we fetch at once in each API call.  But @pettyjamesm was skeptical about whether it even works?  My instinct is this doesn't need a release note, but if it does it should be something like `Improve efficiency of getting partitions from the Glue Metastore by batching requests`.  Would that be accurate? @aweisberg @pettyjamesm what do you think?

Reply:
going with removing the release note.